### PR TITLE
[WIP] README updated

### LINKED ISSUE #78: Add usage instructions to README
A first pass in the README would be great. It should include a few screenshots where helpful but not so many that they're hard to keep up to date.

Some of the information to include:
- Who is it for?
- Typical connection workflows
- Troubleshooting common issues
- Known limitations

### PR DESCRIPTION
**Pull request by lakshyagupta21**
Closes #78

### CODE REVIEW

**Comment by lakshyagupta21** [comment]
Need to add link of apk file.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,132 @@
...
-## Motivation
+Share is an Android application which is a companion app to ODK Collect and it helps different enumerators to share the filled and blanks forms with each other, it also helps organization’s supervisor in reviewing the forms before actually submitting to the server.
 
-There are two use cases which are the main reason for implementing this app
-1. Currently, Collect is used to download blank forms, to fill them and to upload them to the server, there is no collaboration feature yet which help different enumerators to fill the same form with different sections.
-1. Sometimes organizations want to have some review of the forms before sending them to the server. Forms and instances from Collect instead of sending to the server will be sent to this app for the review. Forms from different enumerators of a team will be aggregated to a single device for review.
+### 1.2 Why Share<a name="why-share"></a>
+
+
+Share is designed to enable offline transfer of blank and filled forms. There are three major use cases for this multi-part surveys, supervisor review and blank form distribution.
```
Comment (lognaturel) [inline README.md]: There should be a colon to introduce the list of use cases (for this: multi-part)

File: README.md [hunk 2]
```diff
@@ -1,16 +1,132 @@
...
+
+In some areas, enumerators don’t have any internet connectivity, so they can’t fetch blank forms from the server. Share also allows to send a blank form in case of no internet connectivity provided the devices should be close enough to create a local connection.
+
+## 2. Installation and setup<a name="installation"></a>
+
+The Share app requires ODK Collect installed on the device first because it uses its database and storage for all its forms transfer.
+
+### 2.1 Install and setting up ODK Collect<a name="install-collect"></a>
+
+To install ODK Collect visit this link.
```
Comment (lognaturel) [inline README.md]: Links need to be added in markdown.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,132 @@
...
+## 3. Using Share<a name="using-share"></a>
+Share app can be used by enumerators in sharing forms and by supervisor in reviewing the forms.
+
+### 3.1 Viewing category of forms<a name="view-category"></a>
+
+Launch the app to see  a listing of all distinct versions of blank forms on the home screen. Tapping on any form will open a screen with four different tabs showing the statistics, sent, received and reviewed.
+
+To view details of any form category tap on any category shown on home screen.
+<p align="center">
+  <img src="https://github.com/lakshyagupta21/share/blob/91ad9210c67615263aaccced89616562f77249fa/screenshots/home_screen.png" width="280" height="500"/>
```
Comment (lognaturel) [inline README.md]: These should be relative paths. We can change them later but wanted to make a note of it.

**Pull request merged**